app/views/service_providers/_form.html.erb

```diff
@@ -278,8 +278,11 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
   </div>
   <%= form.input :failure_to_proof_url,
                  input_html: { class: 'usa-input' },
+                 label_html: { class: 'usa-input-required'},
```

Comment:
do we want a conditional on this styling since its only required for IdV?

Reply:
hm, i was going to fix this, but looking at it more closely, i think this is just the red visual indicator. since the `failure_to_proof_url` input only shows up when it's required, i think it's fine to leave it as is, especially since it'll complicate the javascript a bit to include it. what do you think?

Reply:
oh I see, that makes sense. disregard!